### PR TITLE
Fix podman getSubnets()

### PR DESCRIPTION
**Pull request by aojea**
Fix podman getSubnets()

Since we moved to custom networks, it has to get the information from the kind network
It also carries the fix in https://github.com/kubernetes-sigs/kind/pull/2199 that got stuck on CLA signing

### CODE REVIEW

File: pkg/cluster/internal/providers/podman/provider.go [hunk 1]
```diff
@@ -369,6 +369,14 @@ func info(logger log.Logger) *providers.ProviderInfo {
 	info := &providers.ProviderInfo{
 		Rootless: euid != 0,
 	}
+
+	// check if is a local or remote podman
+	// xref: https://github.com/containers/podman/issues/10289
+	err := exec.Command("podman", "--remote", "info").Run()
+	if err == nil {
+		// podman is remote we can't check cgroups information so we exit now
```
Comment (AkihiroSuda) [inline pkg/cluster/internal/providers/podman/provider.go:377]: Alternative PR: https://github.com/kubernetes-sigs/kind/pull/2256

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]

File: pkg/cluster/internal/providers/podman/provider.go (hunk 1 above)
Comment (aojea) [reply pkg/cluster/internal/providers/podman/provider.go:377]: Thanks Akihiro, that is much better

**Comment by BenTheElder** [comment]
what's the context? did the previous one ever work? did podman change something?

**Comment by aojea** [comment]
> what's the context? did the previous one ever work? did podman change something?

that is stuck https://github.com/kubernetes-sigs/kind/pull/2199
and when we moved to custom networks I forget to pass the kind network

> error unrelated
E: Failed to fetch https://packages.microsoft.com/ubuntu/20.04/prod/dists/focal/main/binary-amd64/Packages.bz2  File has unexpected size (73402 != 73160). Mirror sync in progress? [IP: 40.74.238.15 443]
   Hashes of expected file:
    - Filesize:73160 [weak]

**Comment by BenTheElder** [comment]
/lgtm
/approve
I still think json should be high on the list, but /shrug

**Comment by aojea** [comment]
> I still think json should be high on the list, but /shrug

I know I know ... too many things sorry

**Pull request merged**